src/hooks/useForm.tsx

```diff
@@ -140,9 +140,16 @@ export function useForm<TValue, TChangeEvent>({
     if (isFieldError) {
       return (
         <>
-          <WarningIcon color="error" sx={WARNING_ICON_STYLES} />
+          <WarningIcon
+            color="error"
+            sx={{
+              color: 'var(--color-textWarningYellow)',
+              ...WARNING_ICON_STYLES,
+            }}
+          />
           <Typography
             component="span"
+            color="var(--color-textWarningYellow)"
```

Comment:
Can we put this color in the `sx` prop as well like we do above for `WarningIcon`?